README.md

```diff
@@ -134,6 +134,7 @@ validate := validator.New(validator.WithRequiredStructEnabled())
 | Tag | Description |
 | - | - |
 | alpha | Alpha Only |
+| alphaspace | Alpha Space |
```

Comment:
I'm a bit annoyed, this could lead to obesity

I mean, someone will then ask for alphanumspace, alphaunicodespace, and so on

The regexps are computed by the lib even if no one use it, so lib becomes slower and slower.

The PR addresses the issue raised. So it's OK.

But I would like to step out a bit.

The regexp rule idea was already declined, for good reasons

- https://github.com/go-playground/validator/issues/1239

So I think the way to consider doing this is via the custom validator

https://github.com/go-playground/validator/issues/1191#issuecomment-1886729222

Reply:
@ccoVeille 
Thank you for your detailed feedback. I understand and agree your concerns regarding the potential growth and its impact on library performance.

If you feel a custom validator is the better path for this functionality, I’m happy to close this pull request if necessary.

Reply:
I'm a random Gopher,user of the lib

Let's wait for a maintainer's feedback

Reply:
> I'm a random Gopher,user of the lib

oh, I mistakenly thought you were the maintainer.

Reply:
Well, the regex's are lazy loaded, so if anyone doesn't use them, it doesn't affect them as much. 
 
 In this case if people think this is a good addition, I don't think we should block it because we worry about it not being used and slightly bigger binary.

Reply:
Thanks for  your explanation